github-actions/feature-request/src/action.ts

```diff
@@ -0,0 +1,210 @@
+import { GitHubAPI, GitHubIssueAPI, MarkedComment, Issue } from './api';
+import { log } from './log';
+
+export enum CommentMarkers {
```

Comment:
I'd propose to add a quick comment on how these markers are used (we add them into comment body and classify comments later based on these comments).